ec/src/test/kotlin/tech/figure/hdwallet/ec/TestKeyConversion.kt

```diff
@@ -41,6 +41,11 @@ class TestKeyConversion {
         registerBouncyCastle()
     }
 
+    @Test
+    fun testDecodingPublicKeyFromBase64String() {
+        PublicKey.fromString("AmqcgLOp640tgccRYL/+PtKftP0NwHcDNzUiHNsJV+gb").toJavaECPublicKey()
```

Comment:
It's not necessarily required because if there's a problem converting to a `java.security.PublicKey` due to an invalid EC coordinate or something, an exception would be raised. But you're right to point out that a test should have an exception because it should _assert_ some condition as part of the test.